packages/core/types/index.js

```diff
@@ -98,6 +97,16 @@ export type Dependency = {
   meta?: JSONObject
 };
 
+export type Dependency = {
```

Comment:
Wondering if this should be DependencyRequest? Kinda makes sense since it is a combination of a dependency "description" (or whatever we decide to name it) and the site where the dependency was requested from.